packages/host/app/components/operator-mode/profile-info-popover.gts

```diff
@@ -25,6 +26,7 @@ export default class ProfileInfoPopover extends Component<Signature> {
   @service declare matrixService: MatrixService;
   @tracked private isOpened = false;
   @tracked private isRendered = false;
+  @tracked private settingsIsOpenFIXME = false;
```

Comment:
This `FIXME` is because I wasn’t sure the settings component belonged in here but also thought it would be annoying to manage its open/closed state elsewhere as the button to open settings is in this component. Ultimately I’m leaning toward it being fine to have this here because I’m thinking of settings as an expanded version of the popover.

Reply:
I think it' completely fine to have it here.